Fix a CSS bug related to tab menu vertical alignment

div-seungha opened the pull request:
Closes: #232 
Closes: #234

sophie-cluml commented:
We need to keep the title under 50 characters in English. Could you fix it please?

div-seungha commented:
> We need to keep the title under 50 characters in English. Could you fix it please?

I found that there was a problem with my git setting and corrected it. I modified the commit message to less than 50 characters. And as clippy was updated, I pushed the new commit revised the related parts.

sophie-cluml commented:
It seems like you fixed the clippy errors. We need to link the issue. I just assigned you to the issue.

div-seungha commented:
> It seems like you fixed the clippy errors. We need to link the issue. I just assigned you to the issue.

I have just linked the issue!
